Archive/Wiedijk100Theorems/BuffonsNeedle.lean

```diff
@@ -77,21 +77,17 @@ namespace BuffonsNeedle
 variable
   /- Probability theory variables. -/
   {Ω : Type*} [MeasureSpace Ω]
-
```

Comment:
Just a test of the actions.  Please, do not merge based on this!

maintainer merge?